src/SpectatorView.Unity/Assets/SpectatorView/Scripts/SpatialAlignment/WorldAnchor/WorldAnchorCoordinateService.cs

```diff
@@ -73,6 +73,7 @@ public async Task InitializeKnownCoordinatesAsync()
             {
                 GameObject anchorGameObject = new GameObject(knownId);
                 WorldAnchor anchor = store.Load(knownId, anchorGameObject);
+                UnityEngine.Object.DontDestroyOnLoad(anchorGameObject);
```

Comment:
>UnityEngine.Object.DontDestroyOnLoad(anchorGameObject); [](start = 16, length = 55)

This will conflict with the change I already made to parent the anchor to a part of the prefab

Reply:
i will remove this